rosette/api.py

```diff
@@ -1010,3 +1011,11 @@ def similar_terms(self, parameters):
         :return: A python dictionary containing the similar terms and their similarity
         """
         return EndpointCaller(self, self.endpoints['SIMILAR_TERMS']).call(parameters)
+
+    def events(self, parameters):
+        """
+        Topics returns events related to the provided content
```

Comment:
Unclear how this is related to Topics.

Can instead be something like this:

Create an L{EndpointCaller} to identify events found in the texts.
        @param parameters: An object specifying the data,
        and possible metadata, to be processed by the events identifier.
        @type parameters: L{DocumentParameters} or L{str}
        @return: A python dictionary containing the results of event extraction.

Reply:
Thanks! I modified the description by your suggestion.

Reply:
Are you going to make a separate PR request for the examples?

Reply:
Yes, that was the plan. The current example works only with a very specific model, but soon there will be a new, general model. The example sentence must be modified when the new model arrives.